src/Components/GenreSelect/GenreSelect.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { buttonStyle, containerStyle, selectedButtonStyle } from "./styles";
+
+export const GenreSelect = ({ allGenres, selectedGenre, onSelect }) => {
+    const handler = (e) => {
```

Comment:
There is a pattern in react where handlers are usually called `handleClick`, or `handleSelect`, or something similar starting with "handle"; and props are called `onClick`, `onSelect` and so on.

```suggestion
    const handleClick = (e) => {
```